systemtest/src/main/java/io/strimzi/systemtest/utils/ClientUtils.java

```diff
@@ -62,7 +62,7 @@ public static void waitForClientSuccess(String jobName, String namespace, int me
 
     private static long timeoutForClientFinishJob(int messagesCount) {
         // need to add at least 1-2minutes for finishing the job
-        return (long) messagesCount * 1000 + Duration.ofMinutes(2).toMillis();
+        return (long) messagesCount * 1000 + Duration.ofMinutes(3).toMillis();
```

Comment:
Why?

Reply:
In some cases the job needed one more minute on Jenkins -> on local env two minutes are sufficient ...

Reply:
I think that 2 minutes will be enough -> I forgot that I added the `auto.reset.offset.config: earliest` so I'm gonna change it back